src/MagnumPlugins/AssimpImporter/Test/Test.cpp

```diff
@@ -0,0 +1,257 @@
+/*
```

Comment:
What about the `src/MagnumPlugins/AssimpImporter/Test/.Test.cpp.swp` file here? ;)